dashboard/src/reducers/charts.ts

```diff
@@ -24,8 +24,8 @@ const chartsSelectedReducer = (
         error: undefined,
         readmeError: undefined,
         version: action.payload.chartVersion,
-        values: action.payload.values ? action.payload.values : state.values,
```

Comment:
Is this the same as:
```
values: action.payload.values || state.values,
```
? But fine either way.

Reply:
it *was* the same, I changed it to just take into account the payload value